command_line/integrate.py

```diff
@@ -51,6 +51,10 @@
       .type = str
       .help = "The experiments output filename"
 
+    copy_input_reflections = False
```

Comment:
`copy_input_reflections` is incorrect/misleading here, as you are removing reflections that were predicted and extracted from the images, but flagged as not integrated (i.e. there was something "wrong" with the shoebox, e.g. overlapping the edge of a panel, or containing bad pixels etc). It _may_ also remove some input reflections (e.g. unindexed reflections), but I suspect in some cases they will mostly be newly predicted reflections that weren't in the original input.

Reply:
This is countable. I suspect in the cases I was looking at the majority of the reflections I am excluding here _are_ input ones for the simple reason that the reduction in file size was approximately the same as the input reflection file size. However I also recognise that in your recent experience the case could be different.

Reply:
Would `output_unintegrated_reflections` be a more precise option?

Reply:
Believe so so made change...